app/validators/form_password_validator.rb

```diff
@@ -2,8 +2,20 @@ module FormPasswordValidator
   extend ActiveSupport::Concern
 
   included do
+    attr_accessor :password
+    attr_reader :user
+
     validates :password,
               presence: true,
               length: Devise.password_length
+
+    validate :valid_password
+  end
+
+  def valid_password
+    return if errors[:password].any?
+    temp_user = User.new(password: password)
+    return if temp_user.valid?
+    errors.add :password, temp_user.errors[:password]
```

Comment:
Got it, so https://github.com/bitzesty/devise_zxcvbn looks at the contents of the password in terms of words but not length? I would think "weak password" would also include "short password" but I am not very familiar with the lib beyond looking into the README